src/main/java/com/bookstore/bookstoreapi/data/controller/RatingController.java

```diff
@@ -0,0 +1,29 @@
+package com.bookstore.bookstoreapi.data.controller;
+
+import com.bookstore.bookstoreapi.data.entity.Rating;
+import com.bookstore.bookstoreapi.data.service.RatingService;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RatingController {
+
+  @Autowired private RatingService ratingService;
+
+  @PostMapping("/ratings/books")
```

Comment:
I am not sure if you want to have this as `ratings/books`, or `rating/book`, the plural suggests that you are rating multiple.

Reply:
I'll keep it as books, makes sense considering many books can be rated.

Reply:
I wouldn't say so, can this API call rate multiple books at once?